### ISSUE TITLE
texlive-bin: Not pointing to latest version & checksum error

### ISSUE THREAD

**Issue by wallisers** [void-linux/void-packages#21170]
`texlive-bin` is still pointing to `texlive2019-bin`, while the new version `texlive2020-bin` already existis. Also when trying to install `texlive2019-bin` I get a checksum error and the install script aborts.
```
# xbps-install texlive-bin
...
./install-tl: unexpected return value from verify_checksum: -5
```
`texlive2020-bin` and previous versions (2018 and earlier) are working fine.

*xuname: Void 5.4.32 x86_64 GenuineIntel uptodate rFF*
*package: texlive-bin 2019_1, texlive2019-bin 2019_1*

**Comment by svenper** [comment]
* texlive-bin sets the PATH in /etc/profile.d
* install-tl can't find the historic repo yet

these should be fixed by #21090 and #20913

**Comment by wallisers** [comment]
I know. I have read both threads. But `xbps-install -S texlive-bin` still tries to pull 2019_1 on my system.

**Comment by svenper** [comment]
Those pull requests have to be accepted first, and then a while in the build queue as well

**Comment by abenson** [comment]
Fixed in 2b9dbf8ac3e073ed0e81041970ccdaeac2eec21c
